docs/integrations/builtin/app-nodes/n8n-nodes-base.gmail/common-issues.md

```diff
@@ -1,12 +1,12 @@
 ---
 #https://www.notion.so/n8n/Frontmatter-432c2b8dff1f43d4b1c8d20075510fe4
 title: Gmail node common issues 
-description: Documentation for common issues and resolutions in the Gmail node in n8n, a workflow automation platform. Includes details of the issue and suggested resolutions.
+description: Documentation for common issues and questions in the Gmail node in n8n, a workflow automation platform. Includes details of the issue and suggested solutions.
 contentType: integration
 priority: critical
 ---
 
-# Common issues (Gmail node)
+# Gmail node common issues
 
 Here are some common errors and issues with the [Gmail node](/integrations/builtin/app-nodes/n8n-nodes--base.gmail/) and steps to resolve or troubleshoot them.
```

Comment:
```suggestion
Here are some common errors and issues with the [Gmail node](/integrations/builtin/app-nodes/n8n-nodes-base.gmail/) and steps to resolve or troubleshoot them.
```
I know this wasn't a change in the PR, but noticed this link was broken.

Reply:
Ahh, fantastic catch on both counts!